ups/eupspkg.cfg.sh

```diff
@@ -17,20 +17,20 @@ install()
     case $(uname -s) in
         Linux*)
             ana_platform="Linux-x86_64"
-            conda_packages="conda_packages-linux-64.txt"
+            conda_packages="conda3_packages-linux-64.txt"
```

Comment:
How can it break anything if it hasn't existed before? These files come from lsst/lsstsw@09e48c9f97e3ad73cf0436f1155ebf9f8346e0e9 (although changing the python2 files couldn't happen as that would change miniconda3).